.clang-format

```diff
@@ -1 +1,12 @@
-DisableFormat: true
+BasedOnStyle: Google
+UseTab: Never
+TabWidth: 4
+AccessModifierOffset: -2
+BreakBeforeBraces: Linux
+# Wider columns can make conditions harder to read
+ColumnLimit: 90
+# Makes maintaining initalizer lists a little easier
+BreakConstructorInitializersBeforeComma: true
+ConstructorInitializerAllOnOneLineOrOnePerLine: true
+# This makes short functions harder to read
```

Comment:
nit: The comment style contradicts the previous style: one explains the fix (little easier), this one explains the problem (harder to read)